server/routes/notification.js

```diff
@@ -0,0 +1,224 @@
+const authMiddleware = require("../middlewares/authMiddleware");
+const ListProductModel = require("../models/listProductModel");
+const NotificationModel = require("../models/notificationModel");
+const { sequelize } = require("../models/productModel");
+const ProductModel = require("../models/productModel");
+const UserListModel = require("../models/userListModel");
+const { PRICE } = require("../utils/enums");
+const router = require("express").Router();
+
+//Make the given notification id read
+router.put("/read/:id", authMiddleware, async (req, res) => {
+  try {
+    const { id } = req.params;
+    const { id: user_id } = req.user;
+
+    await NotificationModel.update(
+      { isRead: true },
+      {
+        where: {
+          id,
+          user_id,
+        },
+      }
+    );
+
+    //update returns only a 1 or a 0 so we create another query to show the client
+    const newNotification = await NotificationModel.findOne({
+      where: {
+        id,
+        user_id,
+      },
+    });
+
+    res.send(newNotification);
+  } catch (error) {
+    res.status(500).send({
+      msg: "Server Error",
+    });
+  }
+});
+
+//Create a notification using a product id given inside the body
+//Will be used by a service
+router.post("/price", async (req, res) => {
+  const trans = await sequelize.transaction();
```

Comment:
`const transaction = await sequelize.transaction();` No need to cut out the few letters. This will help readability